errors/errors.go

```diff
@@ -35,8 +39,10 @@
 	ErrUnsupportedUDSProtocol = errors.New("only unix is supported")
 	// ErrUnsupportedPlatform occurs when running gnet on an unsupported platform.
 	ErrUnsupportedPlatform = errors.New("unsupported platform in gnet")
-	// ErrServerShutdown occurs when server is closing.
-	ErrServerShutdown = errors.New("server is going to be shutdown")
+
+
```

Comment:
🚫 **[golangci-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>File is not `goimports`-ed (goimports)